yb-voyager/cmd/logging.go

```diff
@@ -54,11 +55,12 @@ func InitLogging(logDir string, disableLogging bool) {
 		return
 	}
 	logFileName := filepath.Join(logDir, "yb-voyager.log")
-	f, err := os.OpenFile(logFileName, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644)
-	if err != nil {
-		panic(fmt.Sprintf("Failed to initialise logging: open log file %q: %s", logFileName, err))
+	logRotator := &lumberjack.Logger{
+		Filename:   logFileName,
```

Comment:
how about having a separate folder called `exportDir/logs` instead of filling up `exportDir` with multiple log files